docs/source/components/nodes/pointcloud.rst

```diff
@@ -0,0 +1,154 @@
+PointCloud
+==========
+
+The PointCloud node is essential for 3D spatial data processing, transforming depth maps into point clouds. This node, combined with visualization libraries such as Open3D, facilitates advanced 3D modeling and analysis, supporting applications in robotics, AR/VR, and environmental scanning.
+
+How to Place it
+###############
+
+.. tabs::
+
+  .. code-tab:: py
+
+    pipeline = dai.Pipeline()
+    pointCloud = pipeline.create(dai.node.PointCloud)
+
+  .. code-tab:: c++
+
+    dai::Pipeline pipeline;
+    auto pointCloud = pipeline.create<dai::node::PointCloud>();
+
+
+Inputs and Outputs
+##################
+
+.. code-block::
+
+                 ┌─────────────────────┐
+  inputDepth     │                     │
```

Comment:
we could do `---------` between `inputDepth` and `passthroughDepth`, to visualize the passthrough